tools/shared/rtlRtns.h

```diff
@@ -125,6 +125,7 @@ typedef enum {
   RTE_fracd,
   RTE_fracdx,
   RTE_fracx,
+  RTE_getarga,
```

Comment:
What's the significance of the F90 group versus the FTN group of runtime function symbols? Putting this symbol here names it as `f90_getarga_i8` instead of `fort_getarga_i8`. I am not certain if there is any practical difference; I am just curious since `GETARG` is a Fortran 77 intrinsic and does not appear in later revisions of the language standard.

Reply:
@mleair Perhaps you might know the answer to my question?

Reply:
Mostly this is historical as we used to have F77, HPF, and F90 runtime routines that each had their own prefix of pgftn, pghpf, and pgf90 respectively. I do not believe it matters much anymore what the prefix is. When we open sourced PGF90, we renamed the prefixes to vendor neutral names.